options/options.c

```diff
@@ -715,6 +715,10 @@ const m_option_t mp_opts[] = {
                       0, INT_MAX, ({"auto", -1})),
 #endif
 
+#if HAVE_VAAPI_HWACCEL
+    OPT_STRING("vaapi-device", vaapi_device, M_OPT_FILE),
```

Comment:
I didn't realize that we already had something like this. Maybe it would be ok to have API specific flags for a while while we figure it out. Let's wait for @wm4 to get back to comment.